src/hooks/useSolanaHealth.ts

```diff
@@ -37,47 +24,25 @@ const useSolanaHealth = () => {
   }, [connection])
 
   useAsync(async () => {
-    const result = await axios(`https://ping.solana.com/${cluster}/last6hours`)
-
-    if (result.data.length) {
-      // Date 15 mins ago
-      const now = new Date()
-      now.setMinutes(now.getMinutes() - 15)
-
-      // Only get results in the last 10 mins
-      const last15Mins = result.data.filter(
-        (item: PingItem) => new Date(item.ts) > now,
-      )
-
-      // Find the highest mean_ms
-      const highestMean = last15Mins.reduce(
-        (prev: PingItem, current: PingItem) =>
-          prev.mean_ms > current.mean_ms ? prev : current,
-      )
-
-      // If the highest mean_ms is greater than 10000ms, set unhealthy
-      if (highestMean.mean_ms > 10000) {
-        setIsHealthy(false)
-        setSlowPing(highestMean)
-      }
+    if (!anchorProvider) return
+    const currentTps = await getCurrentTPS(anchorProvider)
+    // If the highest mean_ms is greater than 10000ms, set unhealthy
```

Comment:
This comment changed right? Its now checking for tps  less than 2000?

Reply:
Ahh good catch. Let me update